docs/dictionary/command/ask.lcdoc

```diff
@@ -27,26 +27,44 @@ ask myPrompt as sheet
+ask tPromptText
+
 Parameters:
-iconType (enum): The icon that is displayed on the left side of the dialog box. If you do not specify an icon, none is displayed. iOS and Android do not support <iconType>.
+iconType (enum): The icon that is displayed on the left side of the dialog box. If you do not specify an icon, none is displayed. 
+iOS and Android do not support <iconType>.
 - information : Non-critical notification
 - question : Request for information from the user
 - error : Notification of error or failure condition
 - warning : Notification of unexpected or abnormal condition
-question (string): A string of formatted (or unformated) text.
-defaultResponse (string): A string, and is placed in the text box when the dialog box appears. If no defaultResponse is specified, the text box is empty when the dialog box appears.
+prompt (string): A string of formatted (or unformatted) text.
+defaultResponse (string): A string, and is placed in the text box when the dialog box appears. If no defaultResponse is specified, 
+the text box is empty when the dialog box appears.
 windowTitle: Appears in the title bar of the dialog box. If no windowTitle is given, the title bar is blank.
 
-The result: If the user cancels the <dialog box|dialog>, the <it> <variable> is set to empty and the <result> <function> <return|returns> "cancel".
+The result: If the user cancels the <dialog box|dialog>, the <it> <variable> is set to empty and the <result> <function> 
+<return|returns> "cancel".
 
 It: The contents of the text box is placed in the it <variable>.
 
 Description:
 Use the <ask> <command> when a <handler> needs to get information from the user before continuing.
 
-If the ask...as sheet form is used, the dialog box appears as a sheet on OS X systems. On other systems, the as sheet form has no effect and the dialog box appears normally. Attempting to open a sheet from within another sheet displays the second stack as a <modal dialog box> instead.
+If the ask...as sheet form is used, the dialog box appears as a sheet on OS X systems. On other systems, the as sheet form 
+has no effect and the dialog box appears normally. Attempting to open a sheet from within another sheet displays the second 
+stack as a <modal dialog box> instead.
+
+You can format the prompt text using <HTML>-style tags as described in the <HTMLText> property.
 
->*Cross-platform note:* On <OS X|OS X systems>, there is no image for the question icon. Therefore, the information icon appears instead. In addition, the image specified by the <gRevAppIcon keyword> appears if you do not specify an <iconType>. If you specify an iconType, the image specified by the <gRevSmallAppIcon keyword> appears instead, along with the standard icon specified by the <iconType>. 
+>*Cross-platform note:* On <OS X|OS X systems>, there is no image for the question icon. 
+Therefore, the information icon appears instead. In addition, the image specified by the <gRevAppIcon keyword> appears 
```

Comment:
Can you make this `<gRevAppIcon>` (i.e. remove the keyword bit) please - that should fix the link too! And similarly for `<gRevSmallAppIcon>` below.